From 96de5aa8dcb61b2b237380dc3859ecf9a0d299c9 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Thu, 23 Nov 2017 17:21:30 +0800
Subject: [PATCH 0670/1051] arm64: dts: marvell: align Armada7040 default dts
 to setup A DB board

This Armada7040 default dts is aligned to setup A DB board topology.
Besides of the CP0 comphy setting, setup A DB board has following
IOs: AP_UART, AP_eMMC, CP0_I2C0, CP0_I2C1, CP0_SPI, CP0_eMMC, CP0_RGMII1,

Change-Id: I15964571c03a07d61b2543af1ccf3141b250481a
Signed-off-by: Victor Gu <xigu@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../arm64/boot/dts/marvell/armada-7040-db.dts | 60 ++-----------------
 1 file changed, 4 insertions(+), 56 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 5c1e87f12118..3741d0e37516 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -9,9 +9,10 @@
 #include "armada-7040-db.dtsi"
 
 / {
-	model = "Marvell Armada 7040 DB board";
-	compatible = "marvell,armada7040-db", "marvell,armada7040",
-		     "marvell,armada-ap806-quad", "marvell,armada-ap806";
+	model = "Marvell Armada 7040 development board default (A) setup";
+	compatible = "marvell,armada7040-db-default", "marvell,armada7040-db",
+		     "marvell,armada7040", "marvell,armada-ap806-quad",
+		     "marvell,armada-ap806";
 
 	chosen {
 		stdout-path = "serial0:115200n8";
@@ -35,29 +36,6 @@
 
 &spi0 {
 	status = "okay";
-
-	spi-flash@0 {
-		#address-cells = <1>;
-		#size-cells = <1>;
-		compatible = "jedec,spi-nor";
-		reg = <0>;
-		spi-max-frequency = <10000000>;
-
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			partition@0 {
-				label = "U-Boot";
-				reg = <0 0x200000>;
-			};
-			partition@400000 {
-				label = "Filesystem";
-				reg = <0x200000 0xce0000>;
-			};
-		};
-	};
 };
 
 &uart0 {
@@ -104,36 +82,6 @@
 	pinctrl-0 = <&nand_pins>, <&nand_rb>;
 	pinctrl-names = "default";
 
-	nand@0 {
-		reg = <0>;
-		label = "pxa3xx_nand-0";
-		nand-rb = <0>;
-		nand-on-flash-bbt;
-		nand-ecc-strength = <4>;
-		nand-ecc-step-size = <512>;
-
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			partition@0 {
-				label = "U-Boot";
-				reg = <0 0x200000>;
-			};
-
-			partition@200000 {
-				label = "Linux";
-				reg = <0x200000 0xe00000>;
-			};
-
-			partition@1000000 {
-				label = "Filesystem";
-				reg = <0x1000000 0x3f000000>;
-			};
-
-		};
-	};
 };
 
 &cp0_spi1 {
-- 
2.17.1

